proto/tsopb.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+package tsopb;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+import "pdpb.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+service Tso {
```

Comment:
This service is to provide TSO storage. Shall we call it TsoStorage to differentiate it from the TSO service which provide GetTSs?